packages/roosterjs-editor-dom/lib/table/tableFormatInfo.ts

```diff
@@ -9,7 +9,7 @@ const TABLE_STYLE_INFO = 'roosterTableInfo';
  * @param table The table that has the info
  */
 export function getTableFormatInfo(table: HTMLTableElement) {
-    const obj = safeParseJSON(table?.dataset[TABLE_STYLE_INFO]) as TableFormat;
+    const obj = safeParseJSON(table?.dataset[TABLE_STYLE_INFO]) as Required<TableFormat>;
```

Comment:
you can change line 28 to be 
```ts
function checkIfTableFormatIsValid(format: any): format is Required<TableFormat> {
```

and do the type cast inside that function, then no need to cast here

Reply:
Can you remove this type cast here? And you can change the type of format to be "any" in line 28